modules/@apostrophecms/ui/ui/apos/components/AposCellLabels.vue

```diff
@@ -1,25 +1,36 @@
 <template>
-  <p
+  <div
     class="apos-table__cell-field apos-table__cell-field--labels"
     :class="`apos-table__cell-field--${header.name}`"
   >
-    <span v-if="manuallyPublished && item.modified">
+    <span v-if="manuallyPublished && item.modified && !item.submitted && item.lastPublishedAt">
       <AposLabel
-        label="Active Draft" class="apos-table__cell-field__label"
+        label="Pending Updates" class="apos-table__cell-field__label"
+        tooltip="There are active changes to this document."
+        :modifiers="[ 'is-success', 'is-filled' ]"
+      />
+    </span>
+    <span v-if="item.submitted">
```

Comment:
Seems like it should be a `v-else-if` since they're related.

Reply:
aye